HW02/Ilya_Gribko/ilya_gribko_hw_02_t_01.rb

```diff
@@ -0,0 +1,43 @@
+# Method task_1 should return:
+# * Full text of the first line with an error
+#   (the presence of the substring error in any case)
+# * If the input is text that does not contain the word error,
+#   return an empty string
+
+class Task1
+  def initialize(file_name)
+    @error = false
+    task_1(file_name)
+  end
+
+  def task_1(file_name)
+    error_checking(file_name)
+  end
+
+  private
+
+  def error_checking(file)
+    File.foreach(file) do |f|
+      if error?(f)
```

Comment:
You don't need to create a method for a such short line
```suggestion
      if f.include?('error')
```

Reply:
changed